tools/mocha/src/lib/eventually.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+let _defaultPollingIntervalInMillis = 100;
+
+const sleep = (duration) => {
+  return new Promise((cb) => setTimeout(cb, duration));
+};
+
+const setEventuallyPollingInterval = (pollingInterval) => {
+  _defaultPollingIntervalInMillis = pollingInterval;
+};
+
+const eventually = async (func, pollingInterval = _defaultPollingIntervalInMillis) => {
+  while (true) {
+    try {
+      return await func();
+    } catch (e) {
+      console.log('Eventually failed due to: %o', e.message);
```

Comment:
I am not sure if this should be `console.log`. It produces ugly messages.
A good approach here would be to have also a `timeout` setting. Once the eventually has retried above the `timeout` setting, you could return the last error caught by the `catch` statement.

Reply:
We are going to change this console.log to edebug.
We were discussing to this timeout feature but we decided to rely on mocha timeout. Of course we could implement such one.